eoscostarica/eos-rate
# Add kubernetes deployment

### GH Issue

* adds a script to manage the k8s deployment. refs #399.
* adapts the EOS-rate ssl setup to the k8s installation. refs #399.

### Steps to test

1. execute scripts/deploy.sh ${public_ip}
1. visit eosrate.io
1. you should see the eos rate page

### Checklist
